src/lightning/fabric/loggers/csv_logs.py

```diff
@@ -105,7 +105,7 @@ def log_dir(self) -> str:
         """
         # create a pseudo standard path
         version = self.version if isinstance(self.version, str) else f"version_{self.version}"
-        return os.path.join(self.root_dir, self.name, version)
+        return os.path.join(self._root_dir, self.name, version)
```

Comment:
Are the changes to use the attribute instead of the property still required? If not, you might want to revert them

Reply:
Yes, otherwise PL CSVLogger will use the `self.root_dir` from its definition, and this was the source of the error.